### PR TITLE
feat: fill configs and write to disk on load

### PR DESCRIPTION
**Pull request by MartinBernstorff**

### CODE REVIEW

**Comment by MartinBernstorff** [comment]
Breaking because of unknown issue with `*args`, issue here: https://github.com/explosion/confection/issues/47

@KennethEnevoldsen, do you have experience here? ❤️

**Comment by KennethEnevoldsen** [comment]
I would run the debugger and check what the type actually is. 

Btw. the issue on confection is quite hard to follow. I would create a minimal example.

**Comment by MartinBernstorff** [comment]
The type is a dict shaped like:
```
{'age_filter': {'@preprocessing': 'age_filter', 'min_age': 4, 'age_col_name': 'pred_age'}}}
pydantic.error_wrappers.ValidationError: 1 validation error for ArgModel
```

**Comment by MartinBernstorff** [comment]
Updated the Confection issue with an example :-)

**Comment by KennethEnevoldsen** [comment]
Hmm odd. You can naturally turn off the type-checking (validate=False). Seem like confection resolved in during [filling](https://github.com/explosion/confection/blob/b0e19dade467a57e19efa616ce96d8a405604f05/confection/__init__.py#L762). However, it seems like it shouldn't be an error as it simply resolve it correctly. Might be that there is a bug in either the config or the resolving.

**Comment by MartinBernstorff** [comment]
Yeah, that succeeds on filling, but removes the `*` key as well! Have added more context to the issue.

**Comment by KennethEnevoldsen** [comment]
Added a few additional stuff to the other issues. It seems like it might be a bug.

**Comment by MartinBernstorff** [comment]
I'll close this for now. We can track it in #426.
